src/tool_paramhlp.c

```diff
@@ -490,8 +490,6 @@ ParameterError add2list(struct curl_slist **list, const char *ptr)
     *list = newlist;
   else
     return PARAM_NO_MEM;
-
-  return PARAM_OK;
```

Comment:
What about if `newlist` is non-NULL and the `if` branch is chosen?

Reply:
Yes, thanks.